src/components/Preview/Preview.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react';
+
+import Preview from './Preview';
+
+describe('Preview', () => {
+  describe('when pass the preview props', () => {
+    it('renders on screen the preview', () => {
+      render(<Preview previewText="test" />);
```

Comment:
Pensei melhor aqui revisando esse componente, ficaria melhor assim e ao invés de passarmos o texto via props, passamos via children, muito mais semantico ao mundo real.

```suggestion
      render(<Preview>test</Preview>);
```